extensions/wikia/FounderEmails/events/FounderEmailsDaysPassedEvent.class.php

```diff
@@ -32,14 +32,14 @@ public function process( Array $events ) {
 			if ( empty( $emailController ) ) {
 				continue;
 			}
+			Wikia::initAsyncRequest( $wikiId );
```

Comment:
I think we should be using this in *every* task regardless of whether we know a place where its necessary.  Given how much our code relies on global state, its very likely that one of the following will be true:

* We miss/forgot a place where we've added code relying on this
* There is code that relies on this below some seldom used conditional
* Someone adds code later on, not realizing it executes within a task and assuming implicit use of globals will be ok

In these situations we allow the introduction of possibly difficult to debug issues that can bite us long after we've forgotten about the code we've added.